ldes-server-port-fetch/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fetching/valueobjects/EventStreamProperties.java

```diff
@@ -0,0 +1,4 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fetching.valueobjects;
+
+public record EventStreamProperties(String versionOfPath, String timestampPath) {
```

Comment:
in fetch, this is a record. In the mongo implementation, it's a record. can we align this?